backend/python/app/services/implementations/comment_service.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from flask import current_app
+from sqlalchemy import func
+
+from ...models import db
+from ...models.comment import Comment
+from ..interfaces.comment_service import ICommentService
+
+
+class CommentService(ICommentService):
+    def __init__(self, logger=current_app.logger):
+        self.logger = logger
+
+    def create_comment(self, comment):
+        try:
+            new_comment = Comment(**comment)
+            new_comment.time = datetime.utcnow()
+            new_comment.resolved = False
+            comment_index = 0
+
+            max = db.session.query(func.max(Comment.comment_index)).filter(
+                Comment.story_translation_content_id
+                == comment["story_translation_content_id"]
+            )
+            last_comment = (
+                db.session.query(Comment.comment_index.label("comment_index"))
+                .filter(
+                    Comment.story_translation_content_id
+                    == comment["story_translation_content_id"],
+                    Comment.comment_index == max,
+                )
+                .first()
+            )
+
+            if last_comment is not None:
```

Comment:
is querying for `last_comment` neccessary? could line 37 instead be `comment_index = max + 1`?